### PR TITLE
jhd1313m1_driver: fix '\n' processing

### PR DESCRIPTION
**Pull request by toxeus**
Thanks to the guys at the other table :+1:

### CODE REVIEW

File: platforms/i2c/jhd1313m1_driver.go [hunk 1]
```diff
@@ -145,6 +147,12 @@ func (h *JHD1313M1Driver) Home() (err error) {
 
 func (h *JHD1313M1Driver) Write(message string) (err error) {
 	for _, val := range message {
+		if val == '\n' {
+			if err = h.SetPosition(16); err != nil {
+				return
+			}
+			continue
+		}
 		if err = h.connection.I2cWrite(h.lcdAddress, []byte{0x40, byte(val)}); err != nil {
```
Comment (mattetti) [inline platforms/i2c/jhd1313m1_driver.go]: nitpick, shoudn't we use `LCD_2NDLINEOFFSET` instead of `0x40` in the `I2cWrite` call?

**Comment by mattetti** [comment]
+1 for this PR, ideally we would also refactor a few places where we use `0x40` and switch to using the newly defined constant.

File: platforms/i2c/jhd1313m1_driver.go (hunk 1 above)
Comment (toxeus) [inline platforms/i2c/jhd1313m1_driver.go]: No, I don't think so. In this context it should be `LCD_SETCGRAMADDR`.

**Comment by toxeus** [comment]
> ideally we would also refactor a few places where we use `0x40`

IMHO we should use constants instead of plain values everywhere. Let's clean this up in a dedicated PR :wink:

**Comment by zankich** [comment]
Closing this PR due to https://github.com/hybridgroup/gobot/pull/209

**Pull request closed without merging**